lib/stm32/common/rtc_common_l1f024.c

```diff
@@ -106,6 +106,7 @@ void rtc_set_wakeup_time(uint16_t wkup_time, uint8_t rtc_cr_wucksel)
 	 *    down-counting.
 	 */
 	RTC_WUTR = wkup_time;
+	RTC_CR &= ~(RTC_CR_WUCLKSEL_MASK << RTC_CR_WUCLKSEL_SHIFT);
 	RTC_CR |= (rtc_cr_wucksel << RTC_CR_WUCLKSEL_SHIFT);
```

Comment:
This is probably a place to use an intermediate reg to avoid having to write back multiple times, but it's a minor optimzation, and not something we need to worry about right now. The fix is more important!